packages/subcommands/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@sapphire/subcommands",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework that adds support for subcommands.",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		}
+	},
```

Comment:
Missing register, which will have to do register on Sapphire command and augment types and such. See other plugins for package.json setup.